### PR TITLE
Fix integration failures

### PR DESCRIPTION
**Pull request by jeremybusk**
## Overview
Fixes gitlab-ci integration tests failures due to change in rnode cli contained in this PR.
https://github.com/rchain/rchain/pull/801
I also removed old bash script that would to some integration testing. It was superseded by python one.

### Does this PR relate to an RChain JIRA issue? 
No
### Complete this checklist before you submit the PR
- [x ] This PR contains no more than 200 lines of code, excluding test code.
- [x] This PR meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards).
- [x ] You have someone in mind to assign for review. Make this assignment after submitting the PR.

### Notes
Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else.

### CODE REVIEW

File: scripts/p2p-test-tool.py [hunk 1]
```diff
@@ -414,7 +414,7 @@ def test_repl_load(container):
             cmd = (f"sudo docker run -u root --rm -it -v {repl_node[i]['volume'].name}:{args.rnode_directory} "
                    f"--cpuset-cpus={args.cpuset_cpus} --memory={args.memory} --name {repl_node[i]['name']} "
                    f"--network {args.network} {args.image} "
-                   f"--grpc-host {container.name} -r")
+                   f"--grpc-host {container.name} repl")
```
Comment (KentShikama) [inline scripts/p2p-test-tool.py]: Isn't it --repl ?

Comment (KentShikama) [reply scripts/p2p-test-tool.py]: Did we change the API?

Comment (jeremybusk) [reply scripts/p2p-test-tool.py]: they are changing to "express" command or whatever. This is like cisco or other advanced consoles that autocomplete

Comment (jeremybusk) [reply scripts/p2p-test-tool.py]: https://github.com/rchain/rchain/pull/801

Comment (jeremybusk) [reply scripts/p2p-test-tool.py]: eval, repl and deploy demo changed. You also have to add run when starting because it is broken up.

**Pull request merged**